ncs/rad_envelope.yaml.jinja2

```diff
@@ -1,5 +1,10 @@
-{%- set mpi_radio_vendor_name = application['config']['CONFIG_SUIT_MPI_RAD_LOCAL_1_VENDOR_NAME']|default('nordicsemi.com') %}
-{%- set mpi_radio_class_name = application['config']['CONFIG_SUIT_MPI_RAD_LOCAL_1_CLASS_NAME']|default('nRF54H20_sample_rad') %}
+{%- if application is not defined %}
```

Comment:
Why negative logic?
i.e.: `if aaplication is defined` .. `elseif radio is defined`

Reply:
Fixed!